src/test/groovy/edu/ie3/datamodel/io/sink/InfluxDbSinkIT.groovy

```diff
@@ -0,0 +1,218 @@
+/*
+ * © 2020. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+package edu.ie3.datamodel.io.sink
+
+import edu.ie3.datamodel.io.FileNamingStrategy
+import edu.ie3.datamodel.io.connectors.InfluxDbConnector
+import edu.ie3.datamodel.models.StandardUnits
+import edu.ie3.datamodel.models.input.NodeInput
+import edu.ie3.datamodel.models.result.connector.LineResult
+import edu.ie3.datamodel.models.result.system.ChpResult
+import edu.ie3.datamodel.models.timeseries.individual.IndividualTimeSeries
+import edu.ie3.datamodel.models.timeseries.individual.TimeBasedValue
+import edu.ie3.datamodel.models.value.PValue
+import edu.ie3.datamodel.models.voltagelevels.GermanVoltageLevelUtils
+import edu.ie3.util.TimeUtil
+import org.influxdb.dto.Query
+import org.testcontainers.containers.InfluxDBContainer
+import org.testcontainers.spock.Testcontainers
+import spock.lang.Shared
+import spock.lang.Specification
+import tec.uom.se.quantity.Quantities
+
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.time.ZonedDateTime
+
+@Testcontainers
+class InfluxDbSinkIT extends Specification {
```

Comment:
It would be great, if you could add tests as well for `persistIgnoreNested` and `persistAllIgnoreNested`.